accountClient/accountClient_test.go

```diff
@@ -158,3 +160,58 @@ func TestInternalGetClientAccountReturnsClientsAccountId(t *testing.T) {
 		t.Errorf("Expected account id to be %s, got %s", accountID, account.AccountID)
 	}
 }
+
+func TestInitEmailUpdateWithValidAccountSucceeds(t *testing.T) {
+	// make internal account client (v1)
+	registrationClient := accountClient.New(ValidClientConfig)
+	_, resp, err := test.MakeE3DBAccount(t, &registrationClient, uuid.New().String(), e3dbAuthHost)
+	if err != nil {
+		t.Fatalf("Could not register account %s\n", err)
+	}
+	currentTime := time.Now().Local()
+	// make internal account client (v2)
+	updateClient := accountClient.NewV2(ValidClientConfigV2)
+	newEmailReq := accountClient.InitiateUpdateEmailRequest{
+		AccountID:    resp.Profile.AccountID,
+		CurrentEmail: resp.Profile.Email,
+		NewEmail:     "test" + uuid.New().String() + "@example.com",
+		CreatedAt:    currentTime,
+		CoolOffEnd:   currentTime.Add(time.Hour * 24),
+	}
+	// Make request to post / initiate the  email update
+	response, err := updateClient.InitiateEmailUpdate(testCtx, newEmailReq)
+	if err != nil {
+		t.Fatalf("Failed to initiate email update \n Email Req: (%+v) \n error %+v", newEmailReq, err)
+	}
+	// Verify that the AccountID returned matches the client
+	if response.AccountID.String() != newEmailReq.AccountID {
```

Comment:
more meaningful to check that the new email matches the requested new email and that the current email is still the original email.